docs/topics/yaml-rule-structure-syntax.adoc

```diff
@@ -0,0 +1,520 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: REFERENCE
+[id="yaml-rule-structure-syntax_{context}"]
+= YAML rule structure and syntax
+
+Rules are written in YAML. Each rule consists of metadata, conditions and actions. It instructs the analyzer to take specified actions when given conditions match.
+
+A YAML rules file in {ProductShortName} contains one or more YAML rules.
+
+[id="yaml-rule-metadata_{context}"]
+== Rule metadata
+
+Rule metadata contains general information about the rule. The structure of metadata is as follows:
+
+[source,terminal]
+----
+ruleId: "unique_id" <1>
+labels: <2>
+  # key=value pair
+  - "label1=val1"
+  # valid label with value omitted
+  - "label2"
+  # valid label with empty value
+  - "label3="
+  # subdomain prefixed key
+  - "konveyor.io/label1=val1"
+effort: 1 <3>
+category: mandatory <4>
+----
+<1> The ID must be unique within the ruleset to which the rule belongs.
+
+<2> See below for a description of the label format.
+
+<3> `effort` is an integer value that indicates the level of effort needed to fix this issue.
+
+<4> `category` describes the severity of the issue for migration. The value can be either `mandatory`, `optional` or `potential`. For a description of these categories, see xref:rule-categories_{context}[Rule categories].
+
+[id="yaml-rule-labels_{context}"]
+=== Rule labels
```

Comment:
Don't know whether it's a standard font size, but if not can be increase the font size for Rule labels, Rule categories, Rules Actions, Rule conditions, Logical conditions as they are sub headings
![Uploading Screenshot from 2024-01-11 15-47-06.png…]()

Reply:
@kpunwatk 
I cannot change the font of these headings. But they will look differently when the document is published.

Reply:
Okay, Thanks !